### ISSUE TITLE
Where is the feature labels of NIPS 2003 feature selection challenges

### ISSUE THREAD

**Issue by WYBupup** [agoscinski/EnsembleMethodsForFeatureSelection#1]
Thanks for your project. I'm trying to do an experiment of ensemble feature selection on the data set of the NIPS 2003 feature selection challenge mentioned in your project. However, I could not find the corresponding labels of features of the data set. Could you please tell the where I can find the feature labels of the data set. Thanks!!

**Comment by agoscinski** [comment]
Hello, this project was very long ago so I don't remember it clearly anymore
Here are the data descriptions
https://github.com/agoscinski/EnsembleMethodsForFeatureSelection/blob/master/data_sets.py
And on this website we got some of the datasets, I am not sure if it was all
https://archive.ics.uci.edu/ml/datasets/Dexter
Please let me know if you cannot find a specific one, then I try to find it again

**Comment by WYBupup** [comment]
Thanks for your reply! I have already implemented the experiment on the data set described in the data_set.py of your project. However, I implemented the experiment without the feature labels which indicates whether the feature is relevant or not (the red circle in the image) because I can't find the feature labels file in the data set's website(UCI repository).
![QQ截图20211111215511](https://user-images.githubusercontent.com/61353633/141301749-58e1bea4-e362-4e1c-b4db-a32c66ccbf7b.jpg)
![QQ截图20211111215533](https://user-images.githubusercontent.com/61353633/141301759-a8d35be8-30e8-43f8-b286-7322082786bd.jpg)
![QQ截图20211111215606](https://user-images.githubusercontent.com/61353633/141301764-9e4292ca-15eb-41da-b370-a0f2fd24c4c1.jpg)

And it is said that the feature labels is withheld from the website. (the following image)
![QQ截图20211111215931](https://user-images.githubusercontent.com/61353633/141302277-c442b287-aa3a-4ceb-b6aa-24acf6767287.jpg)


I wonder if you have the access to the labels of features so that I could evaluate the performance of the feature selection algorithm's precision of selecting relevant features like that in artificial data set.
